### PR TITLE
Do not display LSP cannot open channel message when account is disconnected

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR prevents LSP cannot open channel message when account isn't connected yet(where `AccountState.connectionStatus` is `null` or `ConnectionStatus.disconnected`).

### Changelist:
- Use `AccountState.connectionStatus` switch case to display `StatusText` widgets
  - Get `AccountState` & `LspState` from parent widget
- Improve readability in `ReceivableBTCBox` by using `isChannelOpeningAvailable` variable
  - Passing `OpeningFeeParams` to `FeeMessage` when channel opening is available
- Make `_loadingMessage` empty string by default
- Correct typos in
  - `isChannelOpeningAvailiable` variable
  - `annimated_loader_dialog.dart` file name
  
  ### Other changelist:
- Display ReverseSwapsInProgressPage when there are "reverse swaps in progress" 18cc62e777fc26728827ac3aa8c360802a3ee184)
- Beautify SwapErrorMessage 736f3254efd290ea183968c71ff0b30ecb32b98b
- Pass ReverseSwapFeesRequest to fetchReverseSwapFees 21d9d7e85718b9a0361f580cc32112871ffee15c
- Apply breez_bridge -> breez_sdk file renaming 53a22a0ef50bba19db48641ac2fdc77334c4aee9

### CODE REVIEW

**Review by ubbabeck — APPROVED** [review]
LGMT

File: android/app/build.gradle [hunk 1]
```diff
@@ -54,46 +54,14 @@ android {
...
-            signingConfig signingConfigs.debug
-            applicationIdSuffix '.debug'
-            versionNameSuffix '-DEBUG'
             resValue "string", "app_name", "Breez Cloud - Debug"
         }
         release {
-            signingConfig signingConfigs.release
+            // TODO: Add your own signing config for the release build.
+            // Signing with the debug keys for now, so `flutter run --release` works.
+            signingConfig signingConfigs.debug
```
Comment (roeierez) [inline android/app/build.gradle:64]: Are the changes to this file part of the PR?

**Review by roeierez — CHANGES_REQUESTED** [review]
Small comment regarding the build.gradle, not sure if you wanted these changes.
Other than that LGTM.

File: android/app/build.gradle (hunk 1 above)
Comment (erdemyerebasmaz) [reply android/app/build.gradle:64]: They are intentional but was supposed to be on main branch. They revert  
- #623 

changes so we can keep building APK's via CI

**Pull request merged**